src/main.tsx

```diff
@@ -4,7 +4,7 @@ import App from './App'
 import './index.css'
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
-  <React.StrictMode>
+  // <React.StrictMode>
```

Comment:
```suggestion
  // <React.StrictMode>
```
There probably is another way than removing StrictMode